tests/get_or_create/tests.py

```diff
@@ -484,3 +485,35 @@ def lock_wait():
         updated_person = Person.objects.get(first_name='John')
         self.assertGreater(after_update - before_start, timedelta(seconds=0.5))
         self.assertEqual(updated_person.last_name, 'NotLennon')
+
+
+class InvalidCreateArgumentsTests(TestCase):
+    msg = "Thing has no field named "  # u'does_not_exist' for Python 2, 'does_not_exist' for Python 3
```

Comment:
We can fix the annoying u'' prefix discrepancy on Python 2 by using `"'%s'"` in the error message rather than `%r`. Would you care to send a separate PR to try to address that issue? That'll let us clean up places in the test suite such as `tests/annotations/tests.py:        with six.assertRaisesRegex(self, FieldDoesNotExist, "\w has no field named u?'other_rating'")` to use `self.assertRaisesMessage()` which has the advantage of less cleanup when we drop Python 2 compatibility.